### PR TITLE
Son 2159 support C pins

### PR DESCRIPTION
**Pull request by endresonair**

### CODE REVIEW

File: src/adc.rs
Comment (espenalb) [inline src/adc.rs:1]: Nice

File: src/gpio.rs [hunk 1]
```diff
@@ -64,6 +64,10 @@
 use core::marker::PhantomData;
 
 use crate::rcc::ResetEnable;
+// EB: Todo: If making PR upstream, this should only be include for some boards (see pwr.rs)
+use crate::stm32::SYSCFG;
+// EB: Todo: If making PR upstream, check which boards has relevant Pxy_C pins and update
```
Comment (espenalb) [inline src/gpio.rs:69]: Done?

**Review by espenalb — APPROVED** [review]
NIce!

I assume the merged STM changes are OK, did not review them.

File: src/gpio.rs (hunk 1 above)
Comment (endresonair) [reply src/gpio.rs:69]: No, this involves going through the data sheets for all the supported MCUs (there are at least 13) and the package information for each of those (probably 3-5 for each MCU) to see which have these pins (and then add feature flags in the right places), so it is a rather big undertaking...

**Pull request merged**